app/views/auctions/search.html.erb

```diff
@@ -26,8 +26,20 @@
     <div class="margin-block narrow">
         <div class="grid one column">
             <div class="column">
-                <table class="ui table selectable unstackable fixed" id="auctions-table">
-                    <thead>
+                <table class="ui table selectable mobile stackable fixed" id="auctions-table">
+                    <thead class="mobile-show">
```

Comment:
Can we use semantic class name and move logic (I assume it is visible only in mobile, based on the name) to CSS?

Reply:
Renamed to "mobile show", semantic enough I believe. Wouldn't really define deeper selector scope, because it could be heavily referenced from other views as well.